### PR TITLE
Feat/main layout

### PR DESCRIPTION
**Pull request by SikoraKam**

### CODE REVIEW

File: src/components/content/TextAreaComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Box, Textarea } from "@chakra-ui/react";
+
+const TextAreaComponent: React.FC = (props) => {
+  return (
+    <Box my={5}>
+      <Textarea
+        borderWidth={0}
```
Comment (KalitaKonrad) [inline src/components/content/TextAreaComponent.tsx:8]: Isn't borderWidth equal to zero by default?

File: src/components/content/TextAreaComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        placeholder="Dodaj plik aby rozpocząć rozpoznawanie tekstu"
+        resize={"none"}
+        size={"lg"}
+        variant={"outline"}
```
Comment (KalitaKonrad) [inline src/components/content/TextAreaComponent.tsx:12]: When passing prop as a string we don't need curly brackets:
e.g. `variant="outline"` or `size="lg"`

File: pages/index.tsx [hunk 3]
```diff
@@ -1,16 +1,46 @@
...
-import { Header, Main, Cards, Footer } from "@components";
+import SideContainer from "@components/sideSection/SideContainer";
+import ContentContainer from "@components/content/ContentContainer";
+import UploadButton from "@components/UploadButton";
+import TextAreaComponent from "@components/content/TextAreaComponent";
+import SideTitleComponent from "@components/shared/SideTitleComponent";
+import PreviewList from "@components/sideSection/PreviewList";
 
 const Home: React.FC = () => {
+  const renderLeftSide = () => (
```
Comment (KalitaKonrad) [inline pages/index.tsx:12]: I guess later on we should refactor these 3 render functions to be a separate components - to make the code cleaner

File: src/components/sideSection/PreviewList.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    alignItems={"center"}
+    flex={1}
+    overflowY={"scroll"}
+    css={{
+      "&::-webkit-scrollbar": {
+        display: "none",
+      },
+    }}
+  >
+    {sitesListMock.map((item) => (
```
Comment (KalitaKonrad) [inline src/components/sideSection/PreviewList.tsx:35]: While mapping each item should have `key` prop passed.

**Review by KalitaKonrad — CHANGES_REQUESTED** [review]
Changes requested

File: src/components/content/TextAreaComponent.tsx (hunk 1 above)
Comment (SikoraKam) [reply src/components/content/TextAreaComponent.tsx:8]: no in Textarea

**Pull request merged**